src/query/scan_manager.c

```diff
@@ -4826,6 +4882,21 @@ scan_close_scan (THREAD_ENTRY * thread_p, SCAN_ID * scan_id)
 
     case S_INDX_SCAN:
       isidp = &scan_id->s.isid;
+
+      if (isidp->outer_table_midxkey_build_domains != NULL)
+	{
+	  for (int i = 0; i < isidp->indx_info->key_info.key_cnt; i++)
+	    {
+	      if (isidp->outer_table_midxkey_build_domains[i])
+		{
+		  tp_domain_free (isidp->outer_table_midxkey_build_domains[i]);
+		  isidp->outer_table_midxkey_build_domains[i] = NULL;
+		}
+	    }
+	  db_private_free_and_init (thread_p, isidp->outer_table_midxkey_build_domains);
+	  isidp->outer_table_midxkey_build_domains = NULL;
```

Comment:
```suggestion
	  db_private_free_and_init (thread_p, isidp->outer_table_midxkey_build_domains);
```
db_private_free_and_init 안에서 NULL로 초기화하고 있습니다.

Reply:
https://github.com/CUBRID/cubrid/pull/6661/commits/782a7fa6c5f2a6a84bdc26a5efff2dcb2d622ba1 에서 적용했습니다.